jenkins/ci.suse.de/cloud-mkphyscloud-qa-rally-test.yaml

```diff
@@ -0,0 +1,101 @@
+- job:
+    name: cloud-mkphyscloud-qa-rally-test
+    node: cloud-mkphyscloud-gate-qa
+    description: |
+      Deploy vm on qa hw.
+      Setup rally envirment.
+      Run test
+      Mandatory parameter: hw_number
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    wrappers:
+      - build-name:
+          name: '#${BUILD_NUMBER} - ${scenario_name} - qa$hw_number - full'
+      - timestamps
+    publishers:
+      - mkphyscloud-qa-common-publishers
+
+    concurrent: true
+
+    parameters:
+      - string:
+          name: hw_number
+          default: "2"
+          description: Mandatory, number of the QA cloud server
+
+      - string:
+          name: image_name
+          default: jeos-64
+          description: | 
+            Pre condition to have jeos image uploaded before running rally tests
+
+      - string:
+          name: scenario_name
+          description: Optional; scenario name which typically is an integer with a single letter
+
+      - string:
+          name: scenario_job_name
+          description: Optional; name of the scenario jenkins job that is used to trigger this job
+
+      - string:
+          name: scenario_build_number
+          description: Optional; scenario build number that triggered this job
+
+      - string:
+          name: rally_server
+          default: backup.cloudadm.qa.suse.de
+          description: Server hostname where rally is setup
+
+    builders:
+      - shell: |
+          #!/bin/bash
+          admin=crowbar$hw_number;
+          cloud=qa$hw_number;
+
+          # Create artifacts dir
+          export artifacts_dir=$WORKSPACE/.artifacts
+          rm -rf $artifacts_dir
+          mkdir -p $artifacts_dir
+          touch $artifacts_dir/.ignore
+
+          # Connect to rally server and run tests
+          ssh -T root@$rally_server "bash -s $cloud $image_name" <<'EOF'
+
+          cloud=$1
+          image_name=$2
+
+          source .openrc
+          # Check if jeos image already exist on cloud
+          openstack image show -c name --format value $image_name
+          image_exist=$?
+          # Upload jeos image
+          if [[ $image_exist != 0 ]]; then
+              curl -s http://clouddata.cloud.suse.de/images/jeos-64.qcow2 |
```

Comment:
We should use this repo for jeos image downloading http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/JeOS/images/SLES12-SP2-JeOS-for-OpenStack-Cloud.x86_64-1.2.0-Build13.308.qcow2